internal/monitors/processlist/processlist.go

```diff
@@ -0,0 +1,114 @@
+package processlist
+
+import (
+	"bytes"
+	"compress/zlib"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"math"
+	"runtime"
+	"time"
+
+	"github.com/signalfx/golib/event"
+	"github.com/signalfx/signalfx-agent/internal/core/config"
+	"github.com/signalfx/signalfx-agent/internal/monitors"
+	"github.com/signalfx/signalfx-agent/internal/monitors/types"
+	"github.com/signalfx/signalfx-agent/internal/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	monitorType = "processlist"
+	version     = "0.0.30"
+)
+
+// MONITOR(processlist): This monitor reports processlist information for Windows
+// Hosts.
+//
+// Sample YAML configuration:
+//
+// ```yaml
+// monitors:
+//  - type: processlist
+// ```
+
+var logger = log.WithFields(log.Fields{"monitorType": monitorType})
+var zlibCompressor = zlib.NewWriter(&bytes.Buffer{})
+
+// Config for this monitor
+type Config struct {
+	config.MonitorConfig `singleInstance:"true" acceptsEndpoints:"false"`
+}
+
+func init() {
+	monitors.Register(monitorType, func() interface{} { return &Monitor{} }, &Config{})
+}
+
+// compresses the given byte array
+func compressBytes(in []byte) (buf bytes.Buffer, err error) {
+	zlibCompressor.Reset(&buf)
+	_, err = zlibCompressor.Write(in)
+	_ = zlibCompressor.Close()
+	return
+}
+
+// toTime returns the given seconds as a formatted string "min:sec.dec"
+func toTime(secs float64) string {
+	minutes := int(secs) / 60
+	seconds := math.Mod(secs, 60.0)
+	dec := math.Mod(seconds, 1.0) * 100
```

Comment:
This is quite suspect -- the remainder of `seconds/1.0` would always be 0 right since 1 is a factor of all real numbers.  Did you mean to use `math.Modf`?